test/unit/steps/identity/appointee-form-download/AppointeeFormDownload.test.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const AppointeeFormDownload = require('steps/identity/appointee-form-download/AppointeeFormDownload');
+const {expect} = require('test/util/chai');
+const paths = require('paths');
+
+describe('AppointeeFormDownload.js', () => {
+
+    let appointeeFormDownloadSut;
```

Comment:
For consistency please remove `Sut` from the class name, we haven't adopted that approach and to be honest it doesn't add any value.

Reply:
done